store/modules/settings/getters.js

```diff
@@ -1,34 +1 @@
-export default {
```

Comment:
Right---I think we're saying the same thing.  I think we should keep these as getters inside the Vuex store so that the components access APIs like `number_of_customer_accounts`, rather than getting the length of an array directly.  You had deleted these getters, so I was trying to say that these types of getters that do something to the state before returning it are valid situtations to use `getters`, and help to decouple the business logic of the components from the way that the data is stored inside Vuex